src/components/CommentsCard.jsx

```diff
@@ -1,7 +1,42 @@
+import { fetchCommentsByArticle } from "../../utils/api";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
 export default function CommentsCard() {
+  const { article_id } = useParams();
+  const [comments, setComments] = useState();
```

Comment:
It would be good to see some initial state here which represents the data type you are expecting from the api call, for example, if the comments are returned in an array, then it would be good to set the initial state as an empty array. This will help us identify and infer as developers, the data type we are expecting to be working with.